cmd/kusk/cmd/root.go

```diff
@@ -25,6 +25,8 @@ THE SOFTWARE.
 package cmd
 
 import (
+	"embed"
```

Comment:
Don't need this as it is imported as `_ "embed"` below.

Reply:
for whichever reason it does. If i don't have them both it won't  compile